src/html-conventions-report.xsl

```diff
@@ -21,10 +21,13 @@
     <xsl:import href="html-conventions-lib/class-attributes-html-conventions.xsl"/>
     <xsl:import href="html-conventions-lib/enumeration-html-conventions.xsl"/>
     <xsl:import href="html-conventions-lib/data-type-html-conventions.xsl"/>
+    <xsl:import href="html-conventions-lib/object-html-conventions.xsl"/>
```

Comment:
The files also have dashes ? I wonder if we shall adopt proper Manning for them as well mmmmm